packages/drivers/odsp-driver/src/odspDocumentServiceFactoryCore.ts

```diff
@@ -94,6 +96,8 @@ export class OdspDocumentServiceFactoryCore implements IDocumentServiceFactory {
                     odspLogger,
                     createNewSummary,
                     cacheAndTracker.epochTracker,
+                    fileEntry,
+                    this.hostPolicy.createNewCaching = true,
```

Comment:
Here you are overriding the host policy. You just need to set it to true in case it is undefined.